CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ Note: A subset of crates received patch releases in the 1.2 stream.
   - Fixed [#3369](https://github.com/unicode-org/icu4x/pull/3369), making datagen call `rustfmt` directly instead of using the `rust-format` dependency
 - `icu_datagen` 1.2.4
   - Remove dependency on `clap`'s `"cargo"` feature to better support non-Cargo users (#3388)
+- `icu_datagen` 1.2.5
+  - Remove runtime dependency on hardcoded compile-time segmenter data paths (#3391)
```

Comment:
Maybe mention the cargo cache? "Remove unconditional usage of files in cargo cache" or something